draft-kazuho-httpbis-priority.md

```diff
@@ -240,6 +240,18 @@ compatible for peers that are unaware of the extended meaning.
 The scheme has a single encoding and set of functionality whether it's
 conveyed via a HTTP header or within a frame.
 
+## Impact on other connections
+
+When multiple connections are open, the priority MAY have impact on the
```

Comment:
For my own benefit at least, I would like to understand what exactly "multiple connections" means in this context. There is a difference to multiple HTTP connections to the same authority (something not recommended for H2 or H3) vs multiple connections to different authorities that all compose a page (but are serviced by the same physical machine) vs just the client having multiple TCP connections open .